### PR TITLE
Add baking and grilling results to the blackbox

### PR DESCRIPTION
**Pull request by dragomagol**
## About The Pull Request
Adds baking and grilling results to the blackbox. Microwaved and crafted foods were being logged here, but the logging messages were lost when we got the grilling and baking components.

## Why It's Good For The Game
More food logging yeehaw

## Changelog
No player-facing changes

### CODE REVIEW

File: code/datums/components/grillable.dm [hunk 1]
```diff
@@ -82,6 +81,9 @@
 		if(original_object.custom_materials)
 			grilled_result.set_custom_materials(original_object.custom_materials)
 
+	if(IS_EDIBLE(grilled_result))
+		SSblackbox.record_feedback("tally", "food_made", 1, grilled_result.type)
```
Comment (Mothblocks) [inline code/datums/components/grillable.dm:85]: Can we move this to a proc or to a define since we use it in two places?

**Review by Mothblocks — CHANGES_REQUESTED** [review]

File: code/datums/components/grillable.dm (hunk 1 above)
Comment (dragomagol) [reply code/datums/components/grillable.dm:85]: I'm not sure what you're asking for here. You want the `if(IS_EDIBLE)` and blackbox logging to be in a proc/define? If so, I don't really see the point in making a proc for two reasonably readable lines of code that're used in two places.

Comment (Mothblocks) [reply code/datums/components/grillable.dm:85]: Eh at the very least can you make "food_made" a define, my concern here is just the two drifting

Comment (dragomagol) [reply code/datums/components/grillable.dm:85]: Gotcha, I can do that

File: code/__DEFINES/logging.dm [hunk 2]
```diff
@@ -65,3 +65,5 @@
 //This is an external call, "true" and "false" are how rust parses out booleans
 #define WRITE_LOG(log, text) rustg_log_write(log, text, "true")
 #define WRITE_LOG_NO_FORMAT(log, text) rustg_log_write(log, text, "false")
+
+#define BLACKBOX_LOG_FOOD_MADE(food) SSblackbox.record_feedback("tally", "food_made", 1, food)
```
Comment (Mothblocks) [inline code/__DEFINES/logging.dm:69]: Can we put this inside __DEFINES/food.dm instead

**Pull request merged**